### DISCUSSION TITLE
Feature Request - back-to-top button

### DISCUSSION THREAD

**Discussion by iju707** [imfing/hextra#97] [Ideas]
Having a back-to-top button would be very helpful in long-form content.
I think it would be better if it is located at the bottom right.

**Comment by LITUATUI** [comment]
The current text solution isn't very good, since it blends with the sidebar text.

An icon is language neutral and more  visible. We could use [unicode](https://symbl.cc/en/2B06/).

**Reply by imfing** [reply]
right now it has an icon at the end. 

and you could remove the text part by modifying the `i18n` file
